core/src/utils/gesture/index.ts

```diff
@@ -141,7 +142,20 @@ export const createGesture = (config: GestureConfig): Gesture => {
     return true;
   };
 
+  const blurActiveElement = () => {
+    /* tslint:disable-next-line */
+    if (typeof document !== 'undefined') {
```

Comment:
If used in a gesture, not sure if this is needed -- I don't think SSR would trigger this

Reply:
talked with adam, going to keep it in - better safe than sorry